code/game/world.dm

```diff
@@ -222,8 +222,11 @@ GLOBAL_VAR_INIT(world_topic_spam_protect_time, world.timeofday)
 			if(input["key"] != config.comms_password)
 				return "Bad Key"
 			else
+				var/prtext = input["announce"]
+				if(findtext(prtext, "Pull request merged by") != 0)
```

Comment:
Someone could open a PR with the name `Pull request merged by` and cheat the system here.